war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1341,14 +1341,26 @@ function rowvgStartEachRow(recursive, f) {
     ++p,
     function (e) {
       e.onclick = helpButtonOnClick;
-      e.tabIndex = 9999; // make help link unnavigable from keyboard
+      // Allow keyboard navigation - help buttons shoulld be accessible
+      e.addEventListerner('keydown', function(event){
+        if (event.key === 'Enter' || event.key=== ' '){
+          event.preventDefault();
+          helpButtonOnClick.call(this);
+        }
+      });
     },
   );
 
   // legacy class name
   Behaviour.specify("A.help-button", "a-help-button", ++p, function (e) {
     e.onclick = helpButtonOnClick;
-    e.tabIndex = 9999; // make help link unnavigable from keyboard
+    // Allow keyboard navigation - help buttons shoulld be accessible
+    e.addEventListerner('keydown', function(event){
```

Comment:
`click` handler already provides keyboard handling, using `Enter` for links and `Space` for buttons, long as you don't override the `tabIndex` of the element. Maybe these custom handlers arenot needed?